flower.rb

```diff
@@ -11,7 +11,26 @@ class Flower < Formula
   depends_on "llvm@4"
   depends_on "python3"
 
+  resource "toposort" do
```

Comment:
Is this really needed for the Flower package? That only calls into `aprotoc` to generate some of its C++ source files. The changes made to `arpc.rb` alone should make it possible to call into `aprotoc` properly, right?

Reply:
Perhaps just the `PYTHONPATH` is required since it will install all of the `pip` packages once `arpc` is installed?

Reply:
I think it's something different. The virtualenv you're setting up for `arpc` will install a wrapper around Python 3, right? If so, we should probably replace regex the `aprotoc` script to no longer use `/usr/bin/env python3`, but the path to the wrapper instead.

That way the `aprotoc` will automatically use the proper Python search paths.

Reply:
There's not virtualenv here. I'm installing the packages to a Homebrew-specific `site-packages` directory then set `PYTHONPATH` so that it loads the package from that directory

Reply:
Still, the same problem applies. There shouldn't be any need to change `flower.rb` in the first place, right? It likely means that `aprotoc` hasn't been fixed in such a way that it can run on its own. You now have to patch all consumers (i.e., `flower.rb`), even though all of this should be an implementation detail of `arpc.rb`.

Reply:
`cloudabi-utils` will not install without _both_ of these changes. So I guess I haven't fixed it properly

Reply:
The modules are installed-- so I guess I could just add `PYTHONPATH` to `flower.rb` and not install the `pip` modules a second time (It's useless in hindsight)